styles.css

```diff
@@ -709,4 +876,53 @@ label {
     position: bottom;
     justify-content: center;
   }
+
+  .card {
+    display: flex;
+    flex-direction: column;
+    width: 90%;
+    height: 95vh;
+    margin: 6.25%;
+    overflow: scroll;
+    z-index: 5;
+    background-color: #fff;
+  }
+
+  .popupFooter {
+    display: flex;
+    flex-direction: row;
+  }
```

Comment:
- [ ] Having made a great effort in your design to rectify the changes requested by the previous reviewer, I noticed your `pop-up` card for the `desktop version` is still not looking like what we have in the Figma design. Kindly check below the highlight of where you need to work on:

  - the card is hitting the wall to the bottom which is not meant to be so, the whole card should be centered as we have in the Figma design
  - the text to the right is hitting the wall kindly give it some padding away from the wall to the right
  - the buttons `see live` and `see source` are not well aligned as seen in the Figma design.
  - the close icon should align properly to the right.

> N.B: the mobile version of the pop-up card works fine, Kindly concentrate on the desktop version

Your Design 👇 

![Anne](https://user-images.githubusercontent.com/78634841/210720150-10a20b27-5885-4add-bfa0-da808ce210aa.png)


Figma design 👇 

<img src="https://user-images.githubusercontent.com/77843050/210662966-dd8eec73-4f26-41a8-9d1a-c7bcbc9c5760.png">